.github/workflows/development-workflow.yml

```diff
@@ -58,20 +58,47 @@ jobs:
             - name: Type Check Project
               run: yarn workspaces foreach -p run typecheck
 
-    testing:
-        name: Testing
+    e2e-testing:
+        name: E2E Testing
         runs-on: ubuntu-latest
         environment: development
+        timeout-minutes: 60
 
         steps:
-            - name: TODO
-              run: echo TODO
+            - name: Setup
+              uses: actions/checkout@v2
+
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: '16'
+
+            - name: Yarn Install
+              run: yarn install
+
+            - name: Prisma Generate
+              run: yarn workspace @app/server generate
+
+            - name: Install Playwright
+              run: yarn workspace @app/client run playwright install --with-deps
+
+            - name: Install Playwright Microsoft Edge
+              run: yarn workspace @app/client run playwright install msedge
```

Comment:
What's the motivation behind installing edge specifically here?  A comment would be helpful on why edge but not other browsers.

Reply:
The other command installs all the other browsers but apparently it doesn't install edge as well so I added that to ensure it was also able to run the tests on edge.

Reply:
Makes sense.  Please add a comment describing that.